### PR TITLE
install prody 2.1.0

### PR DESCRIPTION
**Pull request by jamesmkrieger**
This now installs ProDy 2.1.0 by default

### CODE REVIEW

File: prody2/__init__.py [hunk 1]
```diff
@@ -87,28 +87,43 @@ def addProDyPackage(cls, env, version, default=False):
         installCmd = [cls.getCondaActivationCmd()]
 
         # Activate the new environment
-        installCmd.append('conda activate %s;' % ENV_NAME)
+        installCmd.append('conda activate %s; pip install threadpoolctl;' % ENV_NAME)
```
Comment (MohamadHarastani) [inline prody2/__init__.py:90]: it is better to add this as a requirement to the plugin in the setup. Also, it is better to freeze the version (sometimes they update the version and add a new requirement, which can create library conflicts with other scipion packages)
You may add to the setup as `install_requires=['threadpoolctl==3.1.0'], or create requirements.txt file where all the requirements can be added (it gives better visibility to the requirements as well)
https://github.com/scipion-em/scipion-em-continuousflex/blob/master/setup.py#L149

**Review by MohamadHarastani — APPROVED** [review]

**Review by pconesa — APPROVED** [review]
all test passing. Good to go

**Comment by pconesa** [comment]
Be careful, we meet to distinguish a dependencu5 in the plugin from a
dependency in the third party software. The less dependencies in the plugin
the better

**Comment by jamesmkrieger** [comment]
The software has a soft dependency on it and works fine without it. We just can't control the number of threads used without it. The plugin depends on it more strongly as controlling the number of threads is an important feature there.

**Pull request merged**